src/coreclr/tools/aot/ILCompiler/repro/repro.csproj

```diff
@@ -38,9 +37,6 @@
       <ReproResponseLines Include="--feature:System.Linq.Expressions.CanCreateArbitraryDelegates=false" />
     </ItemGroup>
 
-    <WriteLinesToFile File="$(OutputPath)\compile-with-$(LibrariesConfiguration)-libs.rsp"
-                      Lines="@(ReproResponseLines)"
-                      Overwrite="true"
-                      WriteOnlyWhenDifferent="true" />
+    <WriteLinesToFile File="$(OutputPath)\compile-with-$(LibrariesConfiguration)-libs.rsp" Lines="@(ReproResponseLines)" Overwrite="true" WriteOnlyWhenDifferent="true" />
```

Comment:
I know VS insists on these reformats, but they don't help readability. Could you revert this change and the `Target` change above?